ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/extension/ExtensionDescription.java

```diff
@@ -24,12 +24,11 @@
  * @author <a href="mailto:nzamosenchuk@exoplatform.com">Nikolay Zamosenchuk</a>
```

Comment:
It would be good to remove author email. We agreed to remove such definition of author and leave only First and Last names